### PR TITLE
fix assets precompile that broke during rails 6.1 upgrade

### PR DESCRIPTION
**Pull request by grosser**
fixes https://github.com/zendesk/samson/issues/3948#
nobody noticed because old assets stayed around

/cc @zendesk/compute

### CODE REVIEW

**Comment by adammw** [comment]
worth adding a CI test to ensure precompile works and includes expected plugin files?

File: lib/samson/hooks.rb [hunk 1]
```diff
@@ -110,10 +110,6 @@ def add_decorators
         end
       end
 
-      def add_assets_to_precompile
```
Comment (adammw) [inline lib/samson/hooks.rb:113]: why is this no longer required? rails upgrade does it automatically?

**Review by adammw — APPROVED** [review]

File: lib/samson/hooks.rb (hunk 1 above)
Comment (grosser) [reply lib/samson/hooks.rb:113]: yeah, it's automatic, either because of the new version or because it was in the wrong position before

**Comment by grosser** [comment]
idk how to write that test, so not going to invest more effort

**Pull request merged**